treeherder/model/derived/jobs.py

```diff
@@ -1250,6 +1257,81 @@ def store_job_data(self, data, raise_errors=False):
                 placeholders=coalesced_job_guid_placeholders,
                 executemany=True)
 
+    def store_possible_job_data(self, data, raise_errors=False):
+        """
```

Comment:
My plan was to delete the table every day and create a new one from scratch with the latest allthethings.json, but thinking more about it this sounds wasteful. If there was a way of getting just the daily diff and add/remove rows from the database accordingly it would be much better. I'll try to come up with something.